Source/Details/ASDataController.mm

```diff
@@ -391,7 +391,7 @@ - (void)_insertElementsIntoMap:(ASMutableElementMap *)map
       nodeBlock = [dataSource dataController:self supplementaryNodeBlockOfKind:kind atIndexPath:indexPath];
     }
     
-    ASSizeRange constrainedSize;
+    ASSizeRange constrainedSize = ASSizeRangeUnconstrained;
```

Comment:
yes, that's a sensible default value. If `shouldFetchSizeRanges` is false, that means they're using the async layout and the default value will never be used.